Metrics/start-windows-build.ps1

```diff
@@ -0,0 +1,365 @@
+Param(
+    [Parameter(Mandatory=$false)]
+    [string]$GitURL="https://github.com/dcos/dcos-metrics", 
+    [Parameter(Mandatory=$false)]
+    [string]$Branch="master",
+    [Parameter(Mandatory=$false)]
+    [string]$CommitID,
+    [Parameter(Mandatory=$false)]
+    [string]$ParametersFile="${env:WORKSPACE}\build-parameters.json"
+)
+[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
+$ErrorActionPreference = "Stop"
+
+$globalVariables = (Resolve-Path "$PSScriptRoot\..\global-variables.ps1").Path
+$ciUtils = (Resolve-Path "$PSScriptRoot\..\Modules\CIUtils").Path
+
+Import-Module $ciUtils
+. $globalVariables
+
+$global:PARAMETERS = @{
+    "BUILD_STATUS" = $null
+    "LOGS_URLS" = @()
+    "FAILED_COMMAND" = $null
+}
+function Install-Prerequisites {
```

Comment:
We might potentially remove this if we agree to have a separate script that sets up all the prerequisites.

Reply:
agree